package.json

```diff
@@ -62,6 +62,8 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-vitest": "^0.5.4",
     "fossilize": "^0.3.1",
+    "patch-package": "^8.0.0",
+    "postinstall-postinstall": "^2.1.0",
```

Comment:
Do we need the `postinstall-postinstall` package?

Reply:
I think it is needed according to [the `patch-package` instructions](https://github.com/ds300/patch-package?tab=readme-ov-file#yarn-v1) since [the CI is using yarn 1](https://github.com/getsentry/sentry-wizard/actions/runs/14663735607/job/41153718839?pr=889).

Reply:
Thank you I didn't know that.